tests/drivers/build_all/rtc/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_TEST=y
+CONFIG_TEST_USERSPACE=y
```

Comment:
I believe `CONFIG_TEST=y` and `CONFIG_TEST_USERSPACE=y` can be skipped since they are not used for the build :)

Reply:
No. `CONFIG_TEST` (not to be confused with `CONFIG_ZTEST`) marks the build as a test build (which enables options for making tests easier to debug). `CONFIG_TEST_USERSPACE` conditionally enables userspace if supported, which allows for compilation testing the userspace syscall handlers in `drivers/rtc/rtc_handlers.c`.

Reply:
OK, thanks for the explanation :)